bench/trust_token.cc

```
@@ -0,0 +1,347 @@
+  kFinishIssue,
+  kBeginRedeem,
+  kRedeem,
+};
+
+// NOTE: Fixture is not thread-safe.
+class TrustTokenFixture : public benchmark::Fixture {
+ public:
+  TrustTokenFixture() = default;
+
+ protected:
+  bool PrepareClientAndIssuer(const TRUST_TOKEN_METHOD *method,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)